bin/check_tron_jobs.py

```diff
@@ -63,18 +65,22 @@ def compute_check_result_for_job_runs(client, job, job_content):
     )
     action_run_details = client.action_runs(action_run_id.url, num_lines=10)
 
-    if last_state == "succeeded":
+    if last_state == "succeeded" or last_state == "waiting_run_done":
```

Comment:
Are you sure that you want to return 0 for waiting_run_done? That means if a job fails, the alert will resolve as soon as another run starts. Open to discussion, I'm not sure which is better.

Reply:
On second thought, I think I misunderstood this state. What case is this state supposed to cover?

Reply:
I think it would only get "waiting_run_done" if (1) it's the first job run and it's scheduled but not run yet (runtime is set to future,) or (2) the first job run is running and the second job jun is scheduled. As long as the job has one or more run which state is "failed" or "succeeded", it shouldn't get "waiting_run_done" anymore.

Reply:
Okay, maybe we can call it "waiting_for_first_run" or something?

Reply:
Sure, that does sound better.